### DISCUSSION TITLE
Use schema?

### DISCUSSION THREAD

**Discussion by exptjs** [porsager/postgres#986] [Q&A] [answered]
Please tell me, is it possible to specify the name of the scheme when connecting?
When executing a request with a `nodejs` error, the application crashes. Is it possible for `nodejs` not to fail, but only to show an error?

**Comment by porsager** [comment]
Catch the error ;)

About schema, you are probably looking for the search_path connection property.

**Reply by exptjs** [reply]
`search_path` does not exist in type `Options<{}>`

**Reply by porsager** [reply, marked as answer]
It's a connection property. Just search and you shall find
```js
postgres(..., { connection: { search_path: '...' } })
```
